database/postgres/postgres.go

```diff
@@ -266,13 +269,23 @@ func runesLastIndex(input []rune, target rune) int {
 	return -1
 }
 
+func (p *Postgres) quoteIdentifierWithSchema(name string) string {
```

Comment:
Rename to `quoteIdentifier()` since this may or may not include a schema in the name.